script.js

```diff
@@ -0,0 +1,398 @@
+    nameOne.value = "Player X"
+    nameTwo.value = "Computer"
+    if (player === 'x' && gameOver === false && evt.target.textContent !== 'o') {
+        // player moves
+        evt.target.textContent = 'x'  
+        evt.target.removeEventListener('click', turnsPVC) 
+        evt.target.addEventListener('click', emptyAlert)
+        determineWin();
+
+        // computer moves
+        compMoveFunction()
+
+        status.innerText = `Player X's turn` 
+        determineWin();
+    }
+}
+
+// The below function determines the computer move
+function compMoveFunction() {
+    let compNum = Math.floor(Math.random() * 9) + 1 // generate a random number
+    let compMove = 'game' + compNum.toString()      // concatenate the random number with  'game' and turn into a string to create an id name from our HTML cell elements
+    let newArray = [compMove] // turn the string into an array so we can access it
+    let randomCellId = document.getElementById(newArray[0])
+    // access the HTML cell element associated with the randomly generated cell id
+    if (randomCellId.innerText === '' && gameOver === false) {
+        randomCellId.innerText = 'o'  // fill that cell with an 'o' to signify computer move
+        randomCellId.removeEventListener('click', turnsPVC)
+        randomCellId.addEventListener('click', emptyAlert)
+    } else if (randomCellId.innerText === 'x' || randomCellId.innerText === 'o' && gameOver === false) {
+        compMoveFunction()
+    } else {
+        return randomCellId
+    }
+}
+
+// ************* PVP Functions ********************//
+// The below function gathers player names from the HTML form
+function addNames(evt) {
+    evt.preventDefault()
+    let nameOne = playerOne.value
+    let nameTwo = playerTwo.value
+    status.innerText = nameOne + ' will be X\'s. ' + nameTwo + ' will be O\'s.'
+    startButton.disabled = false
+}
+// The blow function determines the board set up at start of play and reset
+function startGamepvp() {
+    pvpButton.disabled = true
+    nameForm.addEventListener('submit', addNames)
+
+    startButton.addEventListener('click', () => {
+        submit.disabled = true
+        timerStatus = "stopped"
+        document.getElementById("time").innerText = "STARTING GAME COUNTER...."
+        startStop();
+        game1.innerText = ''
+        game1.style.backgroundColor = "white"
+        game2.innerText = ''
+        game2.style.backgroundColor = "white"
+        game3.innerText = ''
+        game3.style.backgroundColor = "white"
+        game4.innerText = ''
+        game4.style.backgroundColor = "white"
+        game5.innerText = ''
+        game5.style.backgroundColor = "white"
+        game6.innerText = ''
+        game6.style.backgroundColor = "white"
+        game7.innerText = ''
+        game7.style.backgroundColor = "white"
+        game8.innerText = ''
+        game8.style.backgroundColor = "white"
+        game9.innerText = ''
+        game9.style.backgroundColor = "white"
```

Comment:
Here's another spot that putting a loop could cut down on the amount of redundancy